allennlp/tests/models/biattentive_classification_network_test.py

```diff
@@ -39,7 +39,7 @@ def test_mismatching_dimensions_throws_configuration_error(self):
         # the embedding dimension from the text_field_embedder.
         params["model"]["encoder"]["input_size"] = 10
         with pytest.raises(ConfigurationError):
-            Model.from_params(self.vocab, params.get("model"))
+            Model.from_params(vocab=self.vocab, params=params.get("model"))
```

Comment:
Might be nice to update some documentation somewhere to say that when calling `from_params` you need to pass arguments by name now.